backend/__tests__/cronJobs.test.ts

```diff
@@ -0,0 +1,122 @@
+import { expect } from 'chai';
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import ComplianceReport from '../src/db/schemas/compliance';
+import { startCronJobs } from '../src/cronJobs';
+import * as removeExpiredDrafts from '../src/cronJobs/removeExpiredDrafts';
```

Comment:
That's quite unusual syntax, what's the purpose of that notation?

Reply:
Importing whole file as an object was the only way I was able to get this working. After this I'm using a function imported from this file and it is easier for me to make use of it. 
I did try to use import is like default or exported method but somehow it was not working with "stub" method. I was able to use "stub" method but it was not counting how many times it was used.